README.md

```diff
@@ -11,6 +11,28 @@ Docs: [![Apache 2.0 License](http://img.shields.io/badge/APACHE2-license-blue.sv
 
 Join in: [![Slack Status](http://slack.projecthydra.org/badge.svg)](http://slack.projecthydra.org/) [![Ready](https://badge.waffle.io/samvera/curation_concerns.svg?label=ready&title=Ready)](http://waffle.io/samvera/curation_concerns)
 
+# Shift in Community Focus
+
+During 2017, the [Samvera community](http://samvera.org/) consolidated [Sufia](https://github.com/samvera/sufia) and [CurationConcerns](https://github.com/samvera/curation_concerns) into [Hyrax](https://github.com/samvera/hyrax):
+
+* [planning notes on consolidation](https://wiki.duraspace.org/pages/viewpage.action?pageId=78161232))
```

Comment:
Pls change link to https://wiki.duraspace.org/pages/viewpage.action?pageId=87461232 per Sufia PR review.